packages/stable/src/__tests__/api/files.int.spec.ts

```diff
@@ -78,30 +118,63 @@ describe.skip('Files integration test', () => {
     expect(file.geoLocation).toEqual(localFileMeta.geoLocation);
   });
 
-  test('validate directoryPrefix format against CDF', async () => {
+  test.skip('validate directoryPrefix format against CDF', async () => {
     await client.files.list({
       filter: {
         directoryPrefix: '/test',
       },
     });
   });
 
-  test('retrieve', async () => {
+  test.skip('retrieve', async () => {
     const [retrievedFile] = await client.files.retrieve([{ id: file.id }]);
     expect(retrievedFile.mimeType).toBe(localFileMeta.mimeType);
     expect(retrievedFile.uploaded).toBeTruthy();
     expect(retrievedFile.sourceCreatedTime).toEqual(sourceCreatedTime);
     expect(retrievedFile.directory).toEqual('/test/testing');
   });
 
-  test('retrieve with non-existent id', async () => {
+  test('retrieve by instance id', async () => {
+    const [retrievedFile] = await client.files.retrieve([
+      { instanceId: fileCdmInstanceId },
+    ]);
+    expect(retrievedFile.instanceId).toEqual(fileCdmInstanceId);
+  });
+
+  test('update by instance id', async () => {
+    const testMetadata = { testKey: 'testVal' };
+    const [retrievedFile] = await client.files.update([
+      {
+        instanceId: fileCdmInstanceId,
+        update: { metadata: { set: testMetadata } },
+      },
+    ]);
+    expect(retrievedFile.instanceId).toEqual(fileCdmInstanceId);
+    expect(retrievedFile.metadata).toEqual(testMetadata);
+  });
+
+  test('download by instance id', async () => {
+    try {
```

Comment:
Rewrite this to use jest's `rejects`: https://jestjs.io/docs/tutorial-async#rejects

Reply:
Done